### ISSUE TITLE
get all records?

### ISSUE THREAD

**Issue by mspanish** [mwager/VanillaStorage#1]
Hey great little library here!  How about a function to grab all records?  Would be a good compliment to nuke :)

thanks very much for the efforts here, works great.

**Comment by mwager** [comment]
Hey @mspanish sorry for the late response. I currently only use this lib to store "bigger" amounts of data under different keys, so I never need to fetch all data ("all rows") at once. I keep track of all my keys manually.

But I think this makes sense though and will leave this issue open, maybe I will need this too in future.

This project is still in an early stage and currently running in production in an app with a very large user base and it's doing a good job so far on a big amount of devices. If you got some time, I would also appreciate any Pull Requests, and if it makes sense I'll probably merge it in (-:

**Comment by mwager** [comment]
See https://github.com/mwager/VanillaStorage/commit/f66dea941a726aadf3a7f7d371a76c187c71f192

Closing for now, feel free to re-open if you find some issues.

**Comment by mspanish** [comment]
Hi sorry for no reply, I had switched to PouchDB but after having a lot of problems trying to compile my app across mobile platforms, I've switched back to VanillaStorage and things seem to be working better. I decided to use something similar, I'm storing my keys using store.js, which turns out to be a better way to manage my data, as my individual data files are pretty big, and traversing them needlessly is a waste of resources.

Thanks for the work!
